reference/tools/ticdc/manage.md

```diff
@@ -0,0 +1,268 @@
+ "pid": 12102    # cdc server pid
+}
+```
+
+### Evict the owner node
+
+{{< copyable "shell-regular" >}}
+
+```shell
+curl -X POST http://127.0.0.1:8300/capture/owner/resign
+```
+
+The above command takes effect only for evicting own nodes.
+
+```
+{
+ "status": true,
+ "message": ""
+}
+```
+
+{{< copyable "shell-regular" >}}
+
+```shell
+curl -X POST http://127.0.0.1:8301/capture/owner/resign
+```
+
+For nodes other than own nodes, executing the above command will have the following error returned.
+
+```
+election: not leader
+```
+
+### Stop replication task
+
+Use the following command to stop a replication task:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+curl -X POST -d "admin-job=1&cf-id=28c43ffc-2316-4f4f-a70b-d1a7c59ba79f" http://127.0.0.1:8301/capture/owner/admin
+```
+
+```
+{
+ "status": true,
+ "message": ""
+}
+```
+
+In the above command:
+
+- `admin-job=1` means to stop the replication task. After the task is stopped, all `processor`s are stopped and exit but the configuration and status of the task are saved and can be recovered from `checkpoint-ts`.
+- `cf-id=xxx` is the ID of `changefeed` that needs operation.
+
+### Resume replication task
+
+Use the following command to resume a replication task:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+curl -X POST -d "admin-job=2&cf-id=28c43ffc-2316-4f4f-a70b-d1a7c59ba79f" http://127.0.0.1:8301/capture/owner/admin
+```
+
+```
+{
+ "status": true,
+ "message": ""
+}
+```
+
+In the above command:
+
+- `admin-job=2` means to resume the replication task from `checkpoint-ts`.
+- `cf-id=xxx` is the ID of `changefeed` that needs operation.
+
+### Delete replication task
+
+Use the following command to delete a replication task:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+curl -X POST -d "admin-job=3&cf-id=28c43ffc-2316-4f4f-a70b-d1a7c59ba79f" http://127.0.0.1:8301/capture/owner/admin
+```
+
+```
+{
+ "status": true,
+ "message": ""
+}
+```
+
+- `admin-job=3` means to delete the replication task. After the interface receives the request, all `processor`s are stopped and the configuration information of the task is cleared. The replication status is reserved. No service is available except for the query.
+- `cf-id=xxx` is the ID of `changefeed` that needs operation.
+
+## Error handling
+
+This section introduces how to handle the error occurred when using TiCDC to replicate data.
+
+### An error occurs when TiCDC replicates statements downstream
+
+When an error occurs when TiCDC executes DDL or DML statements downstream, the replication task is stopped.
+
+- If the error occurs because of downstream anomalies or network jitter, directly resume the replication task;
+- If the error occurs because the downstream is incompatible with SQL, resuming the task will fail. In this situation, you can configure the `ignore-txn-commit-ts` parameter in the replication configuration to skip the transaction at `commit-ts` and resume the task.
```

Comment:
```suggestion
- If the error occurs because the downstream is incompatible with the SQL statement, resuming the task will fail. In this situation, you can configure the `ignore-txn-commit-ts` parameter in the replication configuration to skip the transaction at `commit-ts` and resume the task.
```